### PR TITLE
Consolidate stat var (group) related API

### PR DESCRIPTION
**Pull request by shifucun**
* Change VariableGroups API into BulkVariableGroupInfo API. This is to be consistent with VariableGroupInfo API and Variables API. Basically, "info" API returns the meta info of a node, where "*s" API returns a list of queried nodes.

* Change */variable/group/* to */variable-group/* as this has been confusing when coexist with */variables/* and */variable/ancestor/*. Now it's clear if the API is about an SV or SVG.

### CODE REVIEW

File: proto/v1/info.proto [hunk 1]
```diff
@@ -60,4 +60,19 @@ message VariableGroupInfoRequest {
   // only contain the child variable (group) if it has data for at least one
   // of the entities.
   repeated string constrained_entities = 2;
+}
```
Comment (pradh) [inline proto/v1/info.proto:63]: For my understanding, is there no VariableGroupInfoResponse?

**Review by pradh — APPROVED** [review]

File: proto/v1/info.proto [hunk 2]
```diff
@@ -60,4 +60,19 @@ message VariableGroupInfoRequest {
...
+
+
+message BulkVariableGroupInfoRequest {
+  // The variable group dcids to query for.
+  // If not specified, then return all the variable group info.
```
Comment (pradh) [inline proto/v1/info.proto:68]: nit: If we're using "(optional)" convention, use for this field too.

File: proto/v1/info.proto (hunk 1 above)
Comment (shifucun) [reply proto/v1/info.proto:63]: It returns StatVarGroupNode directly

File: proto/v1/info.proto (hunk 2 above)
Comment (shifucun) [reply proto/v1/info.proto:68]: Done

File: proto/v1/info.proto (hunk 1 above)
Comment (pradh) [reply proto/v1/info.proto:63]: That feels a bit inconsistent with other calls, like VariableInfoResponse, which return node + response?

Also, if we have a VariableGroupInfoResponse, perhaps BulkVariableGroupInfoResponse (like other responses) be a repeated of the former?

Comment (shifucun) [reply proto/v1/info.proto:63]: That's a good point. I was hoping to keep original format but good to make it right at this round.

**Pull request merged**